Dockerfile

```diff
@@ -1,7 +1,13 @@
-# NAME: dclong/rustpython
-FROM dclong/rust
-# GIT: https://github.com/legendu-net/docker-rust.git
+FROM rust:1.66-alpine3.17 as builder
 
-ENV PATH=/root/.cargo/bin:$PATH
-RUN cargo install --git https://github.com/RustPython/RustPython \
-    && /scripts/sys/purge_cache.sh
+WORKDIR /rust-python
```

Comment:
Would you change `/rust-python` to `/workdir`? All of my Docker images uses `/workdir` for consistency.